project/Dependencies.scala

```diff
@@ -72,4 +72,8 @@ object Dependencies extends AutoPlugin {
     val slf4jApi = "org.slf4j" % "slf4j-api" % "1.7.30"
     val typesafeConfig = "com.typesafe" % "config" % "1.4.1"
   }
+
+  object Jackson {
+    val databind = "com.fasterxml.jackson.core" % "jackson-databind" % "2.10.1"
```

Comment:
Is this used anywhere? I think Akka pulls through a Jackson dependency for remote serialization but I'd rather not explicitly depend on it unless we really need

Reply:
getting build failures without it..

Reply:
done